packages/app-builder/src/components/Scenario/AstBuilder/AstBuilderNode/Operand/AggregationLabel.tsx

```diff
@@ -27,47 +25,17 @@ export const AggregationLabel = ({
   labelledAst: LabelledAst;
   viewModel: AggregationEditorNodeViewModel;
 }) => {
-  const { t } = useTranslation(['common', 'scenarios']);
-  const [isOpen, setIsOpen] = useState(false);
-
-  return (
-    <Modal.Root open={isOpen} onOpenChange={setIsOpen}>
-      <Modal.Trigger>
-        <OperandLabel
-          operandLabelledAst={labelledAst}
-          variant="view"
-          tooltipContent={
-            <span className="text-xs">
-              {t('scenarios:view_aggregation.tooltip')}
-            </span>
-          }
-        />
-      </Modal.Trigger>
-      <Modal.Content size="medium">
-        <AggregationLabelModalContent builder={builder} viewModel={viewModel} />
-      </Modal.Content>
-    </Modal.Root>
-  );
-};
```

Comment:
I moved the aggregation description (reading mode) from the hacky modal to the tooltip (so no need to display a tooltip `"Click to view details"` you directly see the variable content).

This change impact :
- editor and viewer
- is necessary to merge in fine all operand label inside a single component
- make the code easier to maintain and remove the only difference between read/edit mode